extern/region-4.9/configure

```diff
@@ -1,6 +1,6 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.63 for region 4.8.
+# Generated by GNU Autoconf 2.63 for region 4.7.
```

Comment:
Why is this saying "region 4.7" (and, more importantly, the PACKAGE_VERSION/STRING lines below also say 4.7)?